ext/sdk/resources/sdk-root/shell/src/resource-templates/ts/scaffolder.ts

```diff
@@ -0,0 +1,212 @@
+import { inject, injectable } from "inversify";
+import { FsService } from "backend/fs/fs-service";
+import { concurrently } from "utils/concurrently";
+import { ResourceTemplateScaffolder, ResourceTemplateScaffolderArgs } from "../types";
+
+@injectable()
+export default class TsScaffolder implements ResourceTemplateScaffolder {
+  @inject(FsService)
+  protected readonly fsService: FsService;
+
+  async scaffold({ request, manifest, resourcePath }: ResourceTemplateScaffolderArgs) {
+    const resourceName = request.assetName;
+    
+    manifest.clientScripts.push('dist/*.client.js');
+    manifest.serverScripts.push('dist/*.server.js');
```

Comment:
Manifest population lacks of watch and build commands, as in `fxdk_watch_command` and so on.

Reply:
Could you describe quick how the `fxdk_watch_command` work? I can't quite figure that one out 😅

Reply:
Yes you need both. One is for rolling rebuilds that get updated w/ changes, and the other is for one off builds (deployments)

`fxdk_watch_command 'yarn' {'{'}'watch', '--mode=development'{'}'}` - yarn is a command, next part is command args.
`fxdk_build_command 'yarn' {'{'}'build'{'}'}` - same as watch command.

Do note that you can specify multiple commands of same type, so you can have multiple watch/build commands even with the same command part.

Reply:
```lua
fxdk_watch_command 'yarn' {'webpack', '--mode', 'development', '--watch'}
fxdk_watch_command 'yarn' {'webpack', '--mode', 'production'}
```

This seems to be working as it should, anything that should be changed?